libpod/image/image.go

```diff
@@ -68,6 +68,17 @@ type Runtime struct {
 	EventsLogFilePath   string
 }
 
+// NewImageConfig is a configuration for a new image to be passed into runtime.New
+type NewImageConfig struct {
```

Comment:
Can we make all the members public, and have a method to return a default ImageConfig, which we can then populate in the caller and pass in? Avoids the long CreateNewImageConfig call

Reply:
fixed